<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget{background-color: rgb(99, 99, 99);}</string>
  </property>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>551</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/mainwnd/add_hover.png);
}
QPushButton:hover
{
border-image: url(:/mainwnd/add_normal.png);
}
QPushButton:pressed
{
border-image: url(:/mainwnd/add_pressed.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>80</y>
     <width>81</width>
     <height>461</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background-color:rgb(99, 99, 99);}
QScrollBar:vertical {
    width:8px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:9px; 
    padding-bottom:9px;
   }
   QScrollBar::handle:vertical {
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;  
    min-height:20;
   }

   QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
      border: none; 
     background: none; 
     color: none; 
   }
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>79</width>
      <height>459</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>78</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget{border-image: url(:/mainwnd/headbg.png);}</string>
   </property>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>918</x>
      <y>7</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mainwnd/min.png);border-style:hidden;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>948</x>
      <y>7</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mainwnd/max.png);border-style:hidden</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>978</x>
      <y>7</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:  url(:/mainwnd/close.png);border-style:hidden</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/mainwnd/businesscn.png);}
QPushButton:pressed
{
border-image: url(:/mainwnd/businesscn.png);
position:relative;
margin-top:5px;
margin-left:5px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/mainwnd/labelmg.png);}
QPushButton:pressed
{
border-image: url(:/mainwnd/labelmg.png);
position:relative;
margin-top:5px;
margin-left:5px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/mainwnd/checkup.png);}
QPushButton:pressed
{
border-image: url(:/mainwnd/checkup.png);
position:relative;
margin-top:5px;
margin-left:5px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>6</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mainwnd/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>55</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white;
font-size:13px;
font-weight:bold;
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string>商家中心</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>513</x>
      <y>56</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white;
font-size:13px;
font-weight:bold;
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string>标签管理</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>57</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white;
font-size:13px;
font-weight:bold;
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string>检查更新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>16</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white;
font-size:30px;
font-weight:bold;
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string>云森客</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_deviceNum">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>601</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:white;
font-family: &quot;Microsoft YaHei&quot;;
font-size:12px;
background-color:transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Account">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>601</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:white;
font-family: &quot;Microsoft YaHei&quot;;
font-size:12px;
background-color:transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Version">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:white;
font-family: &quot;Microsoft YaHei&quot;;
font-size:12px;
background-color:transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_officeWebSite">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>601</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:white;
font-family: &quot;Microsoft YaHei&quot;;
font-size:12px;
background-color:transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>823</x>
     <y>80</y>
     <width>191</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab:selected {
      border-color: #9B9B9B;
      border-bottom-color: #C2C7CB; /* same as pane color */
  }

  QTabBar::tab:!selected {
      margin-top: 2px; /* make non-selected tabs look smaller */
  }

QTabBar::tab {
      background: transparent;
      border: 2px solid #C4C4C3;
      border-bottom-color: #C2C7CB; /* same as the pane color */
      border-top-left-radius: 4px;
      border-top-right-radius: 4px;
      width:70px;
      padding:2px;
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_client">
    <attribute name="title">
     <string>客户资料</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_order">
    <attribute name="title">
     <string>订单信息</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>400</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab:selected {
      border-color: #9B9B9B;
      border-bottom-color: #C2C7CB; /* same as pane color */
  }

  QTabBar::tab:!selected {
      margin-top: 2px; /* make non-selected tabs look smaller */
  }

QTabBar::tab {
      background: transparent;
      border: 2px solid #C4C4C3;
      border-bottom-color: #C2C7CB; /* same as the pane color */
      border-top-left-radius: 4px;
      border-top-right-radius: 4px;
      width:70px;
      padding:2px;
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>个人快捷语</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>公共快捷语</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>80</y>
     <width>741</width>
     <height>505</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>715</width>
     <height>505</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
